spaces/S000129/properties/P000043.md

```diff
@@ -0,0 +1,11 @@
+---
+space: S000129
+property: P000043
+value: true
+refs:
+  - doi: 10.1007/978-1-4612-6290-9
+    name: Counterexamples in Topology
+---
+
+The balls with respect to the metric provided in item #3 for space #132 in {{doi:10.1007/978-1-4612-6290-9}} are naturally arc connected.
+Similarily for the metric defined in the proof of {S129|P55}.
```

Comment:
```suggestion
The balls with respect to the metric provided in item #3 for space #132 in {{doi:10.1007/978-1-4612-6290-9}} are arc connected.
```
Removing the second sentence, as the first one seems perfectly sufficient as justification.  Or am I missing something maybe?